.github/workflows/main.yml

```diff
@@ -158,7 +158,7 @@ jobs:
               tag: ${{ github.ref_name }}
               allowUpdates: true
               removeArtifacts: true
-              artifacts: "nugets/*.nupkg"
+              artifacts: "nugets/*.nupkg;MonoGame.Templates.VSExtension/net472/*.vsix"
```

Comment:
Wrong file path specified for .vsix file.

Reply:
@harry-cpp it picks it up though. What is the right path?

Reply:
vsix/MonoGame.Templates.VSExtension.vsix

Reply:
(this is happening on deploy job, which downloads the artefacts, it builds nothing)

Reply:
Fixed.